projects/wave-core-new/src/lib/operators/dialogs/helpers/layer-selection/layer-selection.component.ts

```diff
@@ -1,10 +1,11 @@
-import {ChangeDetectionStrategy, Component, forwardRef, Input, OnChanges, OnDestroy, SimpleChanges} from '@angular/core';
-import {ControlValueAccessor, NG_VALUE_ACCESSOR} from '@angular/forms';
-import {Layer} from '../../../../layers/layer.model';
-import {BehaviorSubject, Observable, ReplaySubject, Subject, Subscription} from 'rxjs';
-import {ResultType, ResultTypes} from '../../../result-type.model';
-import {ProjectService} from '../../../../project/project.service';
-import {first} from 'rxjs/operators';
+import { ChangeDetectionStrategy, Component, forwardRef, Input, OnChanges, OnDestroy, SimpleChanges } from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { Layer } from '../../../../layers/layer.model';
+import { BehaviorSubject, forkJoin, from, Observable, of, ReplaySubject, Subject, Subscription } from 'rxjs';
+import { ResultType, ResultTypes } from '../../../result-type.model';
+import { ProjectService } from '../../../../project/project.service';
+import { first, mergeMap } from 'rxjs/operators';
+import { LayerMetadata } from 'dist/wave-core/lib/layers/layer-metadata';
```

Comment:
This leads to 

> ERROR: projects/wave-core-new/src/lib/operators/dialogs/helpers/layer-selection/layer-selection.component.ts:8:31 - error TS2307: Cannot find module 'dist/wave-core/lib/layers/layer-metadata' or its corresponding type declarations.
> 
> 8 import { LayerMetadata } from 'dist/wave-core/lib/layers/layer-metadata';
>                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
> 
> 
> Compilation failed.

Reply:
1e26f1695c22100203290979d2e0b01e44efc0ae